source/features/tags-dropdown.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import {getOwnerAndRepo} from '../libs/utils';
+import {octoface} from '../libs/icons';
+
+async function init() {
+	if (select.exists('.blankslate')) {
+		return false;
+	}
+
+	const {ownerName, repoName} = getOwnerAndRepo();
+
+	select('.subnav').append(
+		<div class="rgh-tags-dropdown float-right d-flex flex-shrink-0 flex-items-center mb-3">
+			<details class="details-reset details-overlay select-menu branch-select-menu position-relative">
+				<summary class="btn select-menu-button css-truncate" data-hotkey="w" title="Find tags" aria-haspopup="menu">
+					Select tag&nbsp;
+				</summary>
+				<details-menu
+					class="select-menu-modal position-absolute dropdown-menu-sw"
+					src={`/${ownerName}/${repoName}/ref-list/${getCurrentTag() || 'master'}?source_action=disambiguate&source_controller=files`}
+					preload
+					role="menu"
+					style={{zIndex: 99}}
+				>
+					<include-fragment class="select-menu-loading-overlay anim-pulse" onload={onFragmentLoaded}>
+						{octoface()}
+					</include-fragment>
+				</details-menu>
+			</details>
+		</div>
+	);
+}
+
+function getCurrentTag() {
+	const [tag] = location.href.match(/(?<=\/releases\/tag\/)([^/]+)/) || [undefined];
```

Comment:
```suggestion
	const [tag] = location.href.match(/(?<=\/releases\/tag\/)([^/]+)/) || [];
```

Reply:
Removing `undefined` throws a type error 
```
Initializer provides no value for this binding element and the binding element has no default value
```

Reply:
Method removed as passing tag as a parameter was not making any difference.

Reply:
@sindresorhus we need to decide which one to use:

```js
const [a] = b.match(c) || [undefined];
const [a = undefined] = b.match(c) || [
];
```

I always suggest the former because it feels less noisy than the latter, but as seen in another PR, if the regex captures groups then it’s easier to use `[, a = undefined]`